lib/cluster.js

```diff
@@ -31,14 +31,12 @@ function Worker(options) {
   this.exitedAfterDisconnect = undefined;
 
   Object.defineProperty(this, 'suicide', {
-    get: internalUtil.deprecate(() => {
```

Comment:
@sam-github The deprecation message is not being changed here. 

The new test exercises the getter and setter. They were not being exercised. The code change to `cluster.js` refactor the getter/setter for clarity (good-bye, string concatenation), but do not change the behavior.